### PR TITLE
Adds a fix for syntax problem when generating json by hand

### PR DESCRIPTION
**Pull request by floriank**
This is a small fix for a issue that can pop up when the default case is used
for the first day of the week set in the datepicker via admin settings.

It would have generated the wrong JSON structure breaking _everything_.

Credit goes to @NobodysNightmare for finding this.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://floriank-fix-stupid-stupid-stoopid.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-stupid-stupid-stoopid.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by NobodysNightmare** [comment]
Travis did not find out that this bug was introduced, so I don't care what it says now...

**tl; dr:** merged!
